package.json

```diff
@@ -89,5 +89,8 @@
       }
     },
     "collectCoverage": true
+  },
+  "dependencies": {
+    "sinon": "^7.3.2"
```

Comment:
Always best to pin this ( "sinon": "7.3.2" ) - this avoids unexpected build failures due to updates with bugs and ensures everyone can replicate the same environment. Also put this in devDependencies given we don't actually ship this code to users.